cmd/kubeadm/app/cmd/upgrade/common_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"strings"
 	"testing"
 
-	clientset "k8s.io/client-go/kubernetes"
+	"github.com/spf13/pflag"
```

Comment:
```suggestion
	"github.com/spf13/pflag"

```

empty line.
external / non-k8s libs should be in a separate section under the stdlib section.